src/components/cohortPages/MeetYourTAs.astro

```diff
@@ -10,23 +10,25 @@ const { team, color } = Astro.props;
   class={`flex flex-col rounded-2xl ${color} p-2 lg:gap-10 lg:p-5`}
 >
   <h1
-    class="my-5 text-center font-header text-4xl lg:text-5xl"
+    class="my-3 text-center font-header text-4xl lg:text-5xl"
   >
     Meet Your TAs
   </h1>
 
   {
     team.map((member) => (
-      <div class="my-3 flex rounded-lg bg-black">
+      <div
```

Comment:
Can we use the same component on home page? Its hard to maintain when there are two components that look the same

Reply:
The design will break if we use the same component. It would be better if we make the list constant and use that. But in that case we are not going to use the entire list, so that can cause problem.

Reply:
Try splitting out that component. And maybe make two lists, TA devs and other devs. We can merge them at home page?